gradle/gradle-mvn-push.gradle

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2013 Chris Banes
```

Comment:
We've switched to [gradle-maven-publish-plugin](https://github.com/vanniktech/gradle-maven-publish-plugin) in Okio and it's working well